app-main/public/css/style.css

```diff
@@ -216,7 +216,7 @@ body {
 }
 
 span.lessons-count {
-  margin: 25px;
+  margin:  28px;
```

Comment:
Just altering the margin also throws out the Lesson icon positioning:

![Screen Shot 2020-12-22 at 21 20 12](https://user-images.githubusercontent.com/30320536/102934202-a09a0480-449b-11eb-8a8e-db8784e1c3bd.png)

Could we maybe look at positioning this in a less hard-coded way? For instance, can we use flexbox here? If not, maybe altering the `font-size` will work?

Reply:
Sure !!! I will do necessary changes.